bot/exts/moderation/modpings.py

```diff
@@ -199,17 +216,32 @@ async def on_command(self, ctx: Context) -> None:
         invoke_without_command=True
     )
     @has_any_role(*MODERATION_ROLES)
-    async def schedule_modpings(self, ctx: Context, start: str, end: str) -> None:
-        """Schedule modpings role to be added at <start> and removed at <end> everyday at UTC time!"""
-        start, end = dateutil_parse(start), dateutil_parse(end)
-
+    async def schedule_modpings(self, ctx: Context, start: DayDuration, end: DayDuration) -> None:
+        """
+        Schedule modpings role to be added at <start> and removed at <end> every day at UTC!
+
+        You can have the modpings role off for a maximum of 16 hours i.e. having the modpings role
+        on for a minimum of 8 hours in a day.
+
+        The command excepts two arguments `start` and `end` which represent hour and minute of a day,
```

Comment:
I think this is a typo, you probably want either `accepts` or `expects` here:

```suggestion
        The command accepts two arguments `start` and `end` which represent hour and minute of a day,
```

or

```suggestion
        The command expects two arguments `start` and `end` which represent hour and minute of a day,
```